example/router_example/lib/app/app.dart

```diff
@@ -39,7 +40,7 @@ import 'custom_route_transition.dart';
         page: FavoritesView,
         children: [
           MaterialRoute(page: MultipleFuturesExampleView),
-          CustomRoute(page: HistoryView),
+          //CustomRoute(page: HistoryView),
```

Comment:
Why is `HistoryView` removed?

Reply:
I got an error while building, so I deleted it.